workspace_test.go

```diff
@@ -0,0 +1,55 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestWorkspaceCreation(t *testing.T) {
```

Comment:
Can we split up this test into its own logical segments? I see all of GET, CREATE, UPDATE and DELETE being tested in this one function

Reply:
I can, though if CREATE doesn't work then none of them will work